markhor_flippers/launch/flippers.launch

```diff
@@ -1,9 +1,8 @@
 <launch>
-    <arg name="robot_namespace" default="/markhor/flippers" />
-    
-    <param name="robot_description" command="xacro '$(find markhor_description)/urdf/markhor.urdf.xacro'" />
+    <arg name="robot_namespace" default="/markhor" />
```

Comment:
As discussed, the arg name is misleading and I would rather have the /flippers argument in it. Maybe `flipper_namespace` or something similar would be cleaner?